openeo/extra/job_management.py

```diff
@@ -458,3 +422,60 @@ def ignore_connection_errors(context: Optional[str] = None):
         _log.warning(f"Ignoring connection error (context {context or 'n/a'}): {e}")
         # Back off a bit
         time.sleep(5)
+
+
+class JobTrackerStorage:
```

Comment:
I don't have an alternative yet, but it might be risky to call this "tracker", which is already a thing at the (VITO) backend side

Reply:
@soxofaan, JobDatabasePersistence? In line with Jeroen's APEx documentation: https://jdries-vito.quarto.pub/apex-design/upscaling.html#job-database-persistence